roles/default/kiali-deploy/templates/kubernetes/role.yaml

```diff
@@ -21,6 +21,7 @@ rules:
   - get
   - list
   - watch
+  - patch
```

Comment:
going to need to add all of these to the OLM metadata. I'll add a new github issue for this since we don't have the new manifest directories for the new versions.

Reply:
Need to add "patch" here to the operator role, line 194: https://github.com/kiali/kiali-operator/blob/v1.21/deploy/role.yaml#L194

Reply:
I've added into this PR but you are pointing to a v1.21 version in your comment.
Is this PR to master enough or it is needed to patch in other branch ?

Reply:
This is fine. I only linked to the last release to keep the links from growing stale over time. Just master branch is needed.